src/applications/disability-benefits/526EZ/config/form.js

```diff
@@ -856,10 +863,10 @@ const formConfig = {
                             },
                           ),
                         },
-                        required: [
-                          'providerFacilityName',
-                          'providerFacilityAddress',
-                        ],
+                        // required: [
```

Comment:
Does this commented section need to stay?

Reply:
Thank you! It's been deleted.